cmd/zfs/zfs_main.c

```diff
@@ -3466,9 +3468,31 @@ print_header(list_cbdata_t *cb)
 			(void) printf("%-*s", (int)pl->pl_width, header);
 	}
 
+	color_end();
+
 	(void) printf("\n");
 }
 
+/*
+ * Decides on the color that the avail value should be printed in.
+ * > 80% used = yellow
+ * > 90% used = red
+ */
+static const char *
+zfs_list_avail_color(zfs_handle_t *zhp)
+{
+	int used = zfs_prop_get_int(zhp, ZFS_PROP_USED);
+	int available = zfs_prop_get_int(zhp, ZFS_PROP_AVAILABLE);
```

Comment:
`used` and `available` should be `uint64_t`.   The 'int' in `zfs_prop_get_int()` is misleading...